tests/WebUI/Components/ServiceComponent.test.tsx

```diff
@@ -0,0 +1,110 @@
+import { V1Service } from "@kubernetes/client-node";
+import * as String_Utils from "azure-devops-ui/Core/Util/String";
+import * as Date_Utils from "azure-devops-ui/Utilities/Date";
+import * as React from "react";
+
+import { ServiceComponent } from "../../../src/WebUI/Components/ServiceComponent";
+import * as Resources from "../../../src/WebUI/Resources";
+import { IServiceItem } from "../../../src/WebUI/Types";
+import { mount, shallow } from "../../TestCore";

```

Comment:
as we already have the baseURL, it is better to not have relative paths and rather let the ts compiler resolve using baseUrl

Reply:
as discussed few options likes paths, baseurl did not work, we can take this up later also.